website/docs/docs/advanced-usage/components.md

```diff
@@ -150,6 +150,9 @@ The `<SandpackPreview />` component also allows you to add additional buttons to
   </SandpackLayout>
 </SandpackProvider>
 
+### Additional content
+For advanced use cases, children of `<SandpackPreview>` are rendered at the end of the [preview container](https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/components/Preview/index.tsx#L117).
```

Comment:
This link may become outdated over time, so I'd leave the content as you have now, but without the link.